aws_lambda_powertools/utilities/parser/models/__init__.py

```diff
@@ -50,6 +50,10 @@
     S3Model,
     S3RecordModel,
 )
+from .s3_event_notification import (
+    S3SqsEventNotificationModel,
+    S3SqsEventNotificationRecordModel,
```

Comment:
Hey @theipster, just wanted to give you a heads-up that I changed the class names to make them clearer. I used the prefix `S3Sqs` instead of `SqsS3` since the event comes from S3 to SQS.